packages/design-system/src/components/UsaBanner/UsaBanner.tsx

```diff
@@ -29,123 +29,75 @@ export interface UsaBannerProps {
  */
 export const UsaBanner: React.FunctionComponent<UsaBannerProps> = (props: UsaBannerProps) => {
   const [isBannerOpen, setBannerOpen] = useState<boolean>(false);
-  const [shouldRenderMobileView, setShouldRenderMobileView] = useState<boolean>(false);
   const classes = classNames('ds-c-usa-banner', props.className);
   const id = props.id || uniqueId('gov-banner_');
 
-  useEffect(() => {
```

Comment:
Found a way, using USWDS as a reference, to make this component without the extra JS